matsim/scenariogen/data/__init__.py

```diff
@@ -309,6 +313,7 @@ class Person:
     present_on_day: bool
     reporting_day: int
     n_trips: int
+    home_district: str = ""
```

Comment:
This should belong to the household ?

Reply:
Household already has location and geometry. Is an additional attribute needed?

Reply:
You are right, BUT for the simple routing in the next activity sampling step (because survey data does not provide leg length) this information is needed. It is added to the persons, because I do not want to have to read the whole households.csv in the next step just for one parameter (as the persons / activities datasets already are huge files).

Reply:
Yes, we are talking about 4GB combined only for persons.csv and activities.csv already.. Therefore I cannot run it on my hardware and have to run it on the math cluster, which is annoying for debugging and testing. You have to take into account that we are talking about an area with about 20 million inhabitants, which is way above what we are usually handling (Berlin Brandenburg e.g.)